shared/version/metrics.go

```diff
@@ -0,0 +1,15 @@
+package version
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var prysmInfo = promauto.NewGauge(prometheus.GaugeOpts{
+	Name: "prysm_info",
```

Comment:
```suggestion
	Name: "prysm_version",
```

or `prysm_version_and_committ` is fine too